### ISSUE TITLE
Support for JDK9

### ISSUE THREAD

**Issue by stephenh** [aragozin/jvm-tools#31]
It looks like JDK9 is not supported?

```
ava.lang.ClassCastException: java.base/jdk.internal.loader.ClassLoaders$AppClassLoader cannot be cast to java.base/java.net.URLClassLoader
  at org.gridkit.lab.jvm.attach.AttachAPI.<clinit>(AttachAPI.java:39)
  at org.gridkit.lab.jvm.attach.AttachManager.<clinit>(AttachManager.java:73)
  at org.gridkit.jvmtool.JmxConnectionInfo.getMServer(JmxConnectionInfo.java:72)
  at org.gridkit.jvmtool.cmd.ThreadTopCmd$TTop.run(ThreadTopCmd.java:92)
  at org.gridkit.jvmtool.cli.CommandLauncher.start(CommandLauncher.java:129)
  at org.gridkit.jvmtool.SJK.main(SJK.java:11)
Failed to access MBean server: 22751
```

It makes sense given the classloader changes in Java 9, but I didn't seen an issue open tracking this, so thought I'd open one so I could subscribe to it. :-) Thanks!

**Comment by doom369** [comment]
Same thing with java 10:

```
java -jar sjk-0.10.jar ttop -p 8636 -n 20 -o CPU
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.gridkit.lab.jvm.attach.AttachAPI (file:/user/sjk-0.10.jar) to method java.net.URLClassLoader.addURL(java.net.URL)
WARNING: Please consider reporting this to the maintainers of org.gridkit.lab.jvm.attach.AttachAPI
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
Java home points to /usr/lib/jvm/java-10-oracle make sure it is not a JRE path
Failed to add tools.jar to classpath
java.lang.ClassCastException: java.base/jdk.internal.loader.ClassLoaders$AppClassLoader cannot be cast to java.base/java.net.URLClassLoader
	at org.gridkit.lab.jvm.attach.AttachAPI.<clinit>(AttachAPI.java:39)
	at org.gridkit.lab.jvm.attach.AttachManager.<clinit>(AttachManager.java:73)
	at org.gridkit.jvmtool.JmxConnectionInfo.getMServer(JmxConnectionInfo.java:72)
	at org.gridkit.jvmtool.cmd.ThreadTopCmd$TTop.run(ThreadTopCmd.java:92)
	at org.gridkit.jvmtool.cli.CommandLauncher.start(CommandLauncher.java:129)
	at org.gridkit.jvmtool.SJK.main(SJK.java:11)
Failed to access MBean server: 8636

```

**Comment by aragozin** [comment]
Fixed in 0.10.1
